heroic-core/src/main/java/com/spotify/heroic/shell/task/MetadataEntries.java

```diff
@@ -97,7 +97,7 @@ public TaskParameters params() {
                     io
                         .out()
                         .println(mapper.writeValueAsString(
-                            new AnalyticsSeries(series.getHashCode().toString(),
+                            new AnalyticsSeries(series.getHashCodeTagOnly().toString(),
```

Comment:
this means we will only have analytics around series that are within the metadata index meaning the analytics will be incomplete with regards to series that contain resource identifiers. perhaps this is okay for now and in the future we would want to think about tracking series that contain resource identifiers?  

wdyt @lmuhlha @hexedpackets

Reply:
They were already incomplete so I think it's okay to leave the behavior the same for now. We can iterate on it afterwards.

Reply:
We can also probably remove getHashCode now

Reply:
Removed it, gonna merge now 🎉